docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.7'
+
+services:
+  app: &app
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      args:
+        RUBY_VERSION: '2.3.1'
+        BUNDLER_VERSION: '1.16.6'
+        NODE_MAJOR: '10'
```

Comment:
Can anyone who has access to production confirm that we are using Ruby `2.3.1`, NodeJS `10` and Bundler `1.16.6` there?

Reply:
ruby 2.3.3
bundler 1.16.6
NodeJS 10.25

Reply:
> ruby 2.3.3

Hm, bundler shouldn't allow running the project on any another Ruby version than 2.3.1 since it is specified in `Gemfile` file.